Readme.md

```diff
@@ -50,6 +51,19 @@ Here is an example of defining the PCF85063A in your .overlay
 
 For time set/get you will need to include:
 
+```c
+#include <zephyr/drivers/counter.h>
+#include <drivers/counter/pcf85063a.h>
 ```
-#include <drivers/rtc/pcf85063a.h>
-```
+### Import
+
+You can get the device bindings by name, for example:
+
+```c
+const struct device *rtc;
+
+static void rtc_init() {
+  rtc = device_get_binding("pcf85063a@51");
```

Comment:
Probably best to use `DEVICE_DT_GET` instead since that's the supported path in newer versions of Zephyr.